packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -222,27 +232,24 @@ export const Stackbox = () => {
                     variant="secondary"
                     width="fit"
                     size="xs"
-                    onClick={() => {
-                      setShowTokenAmountError(false);
-                      setShowInsufficentBalanceError(false);
-                      setTokenAmount(
-                        formatUnits(
-                          balance.value / BigInt(divider),
-                          fromToken.decimals
-                        )
-                      );
-                    }}
+                    onClick={() => setTokenAmountBasedOnBalance(divider)}
                   >
                     {name}
                   </Button>
                 ))}
               </div>
               <div className="flex items-center space-x-1">
                 <TokenIcon token={fromToken} size="2xs" />
-                <BodyText className="text-em-high">
-                  <span className="text-em-low">Balance:</span>{" "}
-                  {formattedBalance(balance)}
-                </BodyText>
+                <div className="flex items-center space-x-1">
+                  <BodyText className="flex items-center text-em-low">
+                    Balance:
+                  </BodyText>
+                  <div onClick={() => setTokenAmountBasedOnBalance(1)}>
```

Comment:
Maybe we should have a name for this 1. `Balance.MAX` for example

Reply:
let's name them all.

Reply:
Yes, if we name 1 we name them all :)